### PR TITLE
FIX (duplicates)[#116]: more proper way to check dupes, closes #116

### LINKED ISSUE #116: createNew allows creation of duplicate items because it doesn't check against the existing options...
When the component attempts to create a new item (line 44 of `Dropdown.js`) it makes a call to `valueExistIngInSelected` which checks to see if the value already exists in the `values` property of the internal `state`. 

Whilst this prevents you from having the same item selected twice in the drop down, because it doesn't check the `props.options` list it does not stop you from creating multiple options (there is no way for the component user to protect against this creation as there is no way to feedback from the `createNew` callback that the new item should not be created.

You can repo this very easily in your create new examples because it will give you the option to create a new instance of a person from the existing data.

### PR DESCRIPTION
**Pull request by sanusart**

### CODE REVIEW

**Review by DLatheron — APPROVED** [review]
I've have a play in the code sandbox it created and it looks to solve the issue. Cheers!

File: src/util.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const valueExistInSelected = (value, values, props) =>
-  !!values.find((val) => getByPath(val, props['valueField']) === value);
+  !!values.find((val) => getByPath(val, props['valueField']) === value || val.label === value);
```
Comment (DLatheron) [inline src/util.js:2]: Just a question: shouldn't the access to `val.label` by protected with `getByPath` and using `props['labelField']` too?

Comment (sanusart) [reply src/util.js:2]: @DLatheron you right. It should, will fix and release tonight.

**Review by DLatheron — COMMENTED** [review]
Isn’t it props[‘labelField’]?

**Pull request merged**